homeassistant/components/axis/__init__.py

```diff
@@ -26,8 +26,9 @@ async def async_setup_entry(hass: HomeAssistant, config_entry: ConfigEntry) -> b
     except AuthenticationRequired as err:
         raise ConfigEntryAuthFailed from err
 
-    device = AxisNetworkDevice(hass, config_entry, api)
-    hass.data[AXIS_DOMAIN][config_entry.unique_id] = device
+    device = hass.data[AXIS_DOMAIN][config_entry.entry_id] = AxisNetworkDevice(
+        hass, config_entry, api
+    )
```

Comment:
I think the style was better before.
It would fit on two lines, and I think we try to avoid `x = y = z`

Reply:
Sure!